components/backends/trtllm/performance_sweeps/scripts/bench.sh

```
@@ -0,0 +1,201 @@
+    do_get_logs ${log_path}/output_workers.log ${log_path}/concurrency_${concurrency}
+    echo -n "" > ${log_path}/output_workers.log
+done
+
+
+job_id=${SLURM_JOB_ID}
+if [ -n "${job_id}" ]; then
+    echo "${SLURM_JOB_NODELIST}" > ${log_path}/job_${job_id}.txt
+fi
+
+echo "Benchmark done, gracefully shutting down server and workers..."
+kill -9 $(ps aux | grep '[s]tart_server.sh' | awk '{print $2}') >/dev/null 2>&1 || true
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)